### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MariaSnegireva**
- [DEMO LINK](https://MariaSnegireva.github.io/layout_stars/)
- [TEST REPORT LINK](https://MariaSnegireva.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (Viktor-Kost) [inline src/style.css:30]: redundant as you've already added these rules to .stars__star
```suggestion
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  align-items: center;
+}
+
+.stars__star {
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+
```
Comment (Viktor-Kost) [inline src/style.css:16]:
```suggestion
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (Viktor-Kost) [inline src/style.css:13]: apply background `shorthand`

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (MariaSnegireva) [reply src/style.css]: done

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**